### PR TITLE
[KO] Add `People`

### PR DESCRIPTION
**Pull request by Sitoria**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Main Page
- [x] @sylphi 
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/ko.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- [osu! 팀](osu!_team)
+- [osu! 위키 관리자](osu!_wiki_maintainers)
+
+## 이전
+
+- [비트맵 어프리시에이션 팀](Beatmap_Appreciation_Team)
+- [언어 감시 부서](Language_Surveillance_Division)
+- [매핑 어시스턴트 팀](Mapping_Assistance_Team)
+- [osu!monthly 팀](osu!monthly_team)
+- [Quality Assurance Team](Quality_Assurance_Team)
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/ko.md:32]: 단어장 시트지 업데이트 부탁합니다 그럼 :>>

File: wiki/People/ko.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# 구성원
+
+구성원 또는 그룹을 모아둔 인덱스 페이지입니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/ko.md:3]: d

File: wiki/People/ko.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## 현재
+
+- [어카운트 서포트 팀](Account_support_team)
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/ko.md:7]: ._.)b

File: wiki/People/ko.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- [비트맵 노미네이터](Beatmap_Nominators)
+- [비트맵 스포트라이트 큐레이터](Beatmap_Spotlight_Curators)
+- [센츄리온](Centurions)
+- [커뮤니티 기어자](Community_Contributors)
+- [컴플리셔니스트](Completionists)
+- [개발자](Developers)
+- [엘리트 노미네이터](Elite_Nominators)
+- [공식 아티스트](Featured_Artists)
+- [글로벌 모더레이션 팀](Global_Moderation_Team)
+- [노미네이션 에세스먼트 팀](Nomination_Assessment_Team)
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/ko.md:17]: 얍

File: wiki/People/ko.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+구성원 또는 그룹을 모아둔 인덱스 페이지입니다.
+
+## 현재
+
+- [어카운트 서포트 팀](Account_support_team)
+- [비트맵 노미네이터](Beatmap_Nominators)
+- [비트맵 스포트라이트 큐레이터](Beatmap_Spotlight_Curators)
+- [센츄리온](Centurions)
+- [커뮤니티 기어자](Community_Contributors)
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/ko.md:11]: 아이고 이런 오타났네

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**